doc/man/sconsign.xml

```diff
@@ -64,26 +53,34 @@ Without options,
 individual dependency entries are printed in the following format:</para>
 
 <screen>
-depfile: signature timestamp length
-        implicit_dependency_1: content_signature timestamp length
-        implicit_dependency_2: content_signature timestamp length
+depfile: csig timestamp length
```

Comment:
I think I'd prefer content_signature and build_signature with an explanation of what they are a bit further down than csig and bsig and then explaining later.